src/wr_logic_ai/src/drive_mux_select.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+import rospy
+from wr_logic_ai.msg import NavigationState
+from std_msgs.msg import String
+from typing import Dict
+from topic_tools.srv import MuxSelect
+
+
+def state_call_back(message: NavigationState, conversion_table: Dict[int, str], mux_name : str):
+    rospy.wait_for_service(f"/{mux_name}/select")
+    try:
+        proxy = rospy.ServiceProxy(f"/{mux_name}/select", MuxSelect)
+        proxy(conversion_table[message.state])
```

Comment:
Since the HLSM is going to publish to `/navigation_state` constantly, we probably don't want to constantly call the service.  Should probably switch to an edge-change method.